core/src/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -96,11 +96,6 @@ private string GetTeamId()
                 match = SecKeyChain.QueryAsRecord(queryRecord, out resultCode);
             }
 
-            if (resultCode != SecStatusCode.Success)
```

Comment:
Why are we removing this? Won't we get the null ref because match is null?

Reply:
removing this because this is not the fix. so just going with what we have currently in dev until we find a solution.